### PR TITLE
#57 Implementing Employees list

### PR DESCRIPTION
**Pull request by bkuzmic**
For #57:

- Implementing Employees list view
- Leaving comments and puzzle for further development

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#334](https://github.com/timelesslounge/timelessis/issues/334) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#334](https://github.com/timelesslounge/timelessis/issues/334) is assigned to @vryazanov[/z](https://www.0crat.com/u/vryazanov), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/e7c125ff-6b42-46ea-bac0-b9516d1b53d0); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: tests/integration/it_employees_test.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from http import HTTPStatus
```
Comment (vryazanov) [inline tests/integration/it_employees_test.py]: @bkuzmic small one, by anyway. let's do imports in pep8 way:
```python
from datetime import datetime
from http import HTTPStatus

from timeless.employees.models import Employee
```

File: tests/integration/it_employees_test.py [hunk 2]
```diff
@@ -55,3 +56,17 @@ def create_employee():
...
+def test_list(client, db_session):
+    """ List all employees """
+    employee = Employee(first_name="Alice", last_name="Cooper",
+                        username="alice", phone_number="1", account_status="1",
+                        birth_date=datetime.utcnow(), pin_code=1234,
+                        registration_date=datetime.utcnow(), user_status="1",
+                        email="test@test.com", password="bla")
+    db_session.add(employee)
+    db_session.commit()
+    employees = client.get("/employees/")
```
Comment (vryazanov) [inline tests/integration/it_employees_test.py]: @bkuzmic naming confuses a bit, suggest to rename to `response =`

**Comment by bkuzmic** [comment]
@vryazanov Thanks for the comments. Please take a look now.

File: tests/integration/it_employees_test.py [hunk 3]
```diff
@@ -48,10 +47,21 @@ def test_timestamp_mixin_updated_on(db_session):
...
         pin_code=1234
     )
+
+
+def test_list(client, db_session):
+    """ List all employees """
+    employee = create_employee()
+    db_session.add(employee)
+    db_session.commit()
+    response = client.get("/employees/")
```
Comment (vryazanov) [inline tests/integration/it_employees_test.py]: @bkuzmic One more point I didn't notice before. mb we will use `url_for` instead of hardcoded urls?

Comment (bkuzmic) [reply tests/integration/it_employees_test.py]: @vryazanov Sure, no problem.

**Comment by bkuzmic** [comment]
@vryazanov Now it should be fixed - using url_for instead of hardcoded url.

**Comment by vryazanov** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 6 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job [#334](https://github.com/timelesslounge/timelessis/issues/334) is not in the agenda of @vryazanov[/z](https://www.0crat.com/u/vryazanov), can't inspect

**Comment by 0crat** [comment]
The job [#334](https://github.com/timelesslounge/timelessis/issues/334) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by 0crat** [comment]
Job [#334](https://github.com/timelesslounge/timelessis/issues/334) is pending quality review, can't start a new order

**Comment by ypshenychka** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/timelesslounge/timelessis/issues/334#issuecomment-466182554))

@ypshenychka This is what I know about this job in [CF9LG1YCA](https://www.0crat.com/p/CF9LG1YCA), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#334](https://github.com/timelesslounge/timelessis/issues/334) is not in scope
  * The job is waiting quality review verdict by @ypshenychka[/z](https://www.0crat.com/u/ypshenychka) for 1days
  * @vryazanov[/z](https://www.0crat.com/u/vryazanov) will receive 20 minutes after quality review
  * Job [footprint](https://www.0crat.com/footprint/CF9LG1YCA?q=%7Bjob%3A%27gh%3Atimelesslounge%2Ftimelessis%23334%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by ypshenychka** [comment]
@vryazanov  According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one major issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vryazanov** [comment]
@ypshenychka confirm

**Comment by ypshenychka** [comment]
@vryazanov thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @vryazanov[/z](https://www.0crat.com/u/vryazanov)

**Pull request merged**